chsdi/templates/htmlpopup/schiessanzeigen.mako

```diff
@@ -38,14 +38,14 @@ pdf = 'pdf_%s' % lang
       <table>
     ## # iterating weekdays
       % for i in range(len(c['attributes']['wochentag'])):
-        <tr><td>${_('ch.vbs.schiessanzeigen.wochentag.%s' % c['attributes']['wochentag'][i])}</td>
+        <tr><td>${_('ch.vbs.schiessanzeigen.wochentag.%s' % str(c['attributes']['wochentag'][i]+1))}</td>
```

Comment:
lazy you :-) +1

Reply:
change db schema, update workbench, load data, deploy, change mako, deploy, forget to change model, ...